CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - feat: add option to ignore git changes ([#898](https://github.com/getsentry/sentry-wizard/pull/898))
 - fix(apple): Add additional types to `xcode.d.ts` ([#900](https://github.com/getsentry/sentry-wizard/pull/900))
 - fix: enable debug logs for option `--debug` ([#902](https://github.com/getsentry/sentry-wizard/pull/902))
+- feat: add option to define project directory ([#901](https://github.com/getsentry/sentry-wizard/pull/901))
```

Comment:
The feature is now internal. Maybe it makes sense to remove the changelog entry or to change the entry to reflect that the flag is internal.